www/SimpleCameraPreview.js

```diff
@@ -28,8 +28,18 @@ SimpleCameraPreview.torchSwitch = function (options, onSuccess, onError) {
   exec(onSuccess, onError, PLUGIN_NAME, "torchSwitch", [options]);
 };
 
+SimpleCameraPreview.switchCameraTo = function (options, onSuccess, onError) {
+  options = options || {};
+  options.captureDevice = options.captureDevice || "default";
+  exec(onSuccess, onError, PLUGIN_NAME, "switchCameraTo", [options.captureDevice]);
```

Comment:
```suggestion
  exec(onSuccess, onError, PLUGIN_NAME, "switchCameraTo", [options]);
```

Should it not be all the options here?

Reply:
![Screenshot 2024-04-05 at 16 44 21](https://github.com/spoonconsulting/cordova-plugin-simple-camera-preview/assets/116141944/244dc4f9-225d-4298-b6a2-0181530b0e72)

same as option.flash👀

Reply:
@parveshneedhoo This should also be changed I think as not all options is being pass correctly

Reply:
normally this method takes only one option, what if someone using this plugin passes options others than flash?